modules/authentication/proc-enabling-authentication-with-github.adoc

```diff
@@ -63,10 +63,10 @@ TIP: If you plan to make changes using the GitHub API, ensure that `Read and wri
 `GITHUB_WEBHOOK_URL`:: Enter your {product-short} URL: `pass:c,a,q[{my-product-url}]`.
 `GITHUB_WEBHOOK_SECRET`:: Enter the saved *Webhook secret*.
 
-. To set up the GitHub authentication provider and enable integration with the GitHub API in your {product-short} custom configuration, edit your custom {product-short} ConfigMap such as `app-config-rhdh`, and add the following lines to the `app-config-rhdh.yaml` content:
-.. Configure mandatory fields:
+. To set up the GitHub authentication provider and enable integration with the GitHub API in your {product-short} custom configuration, edit your custom {product-short} ConfigMap such as `app-config-rhdh`, and add the following lines to the `{my-app-config-file}` content:
 +
-.`app-config-rhdh.yaml` fragment with mandatory fields to enable authentication with GitHub
+--
+.`{my-app-config-file}` fragment with mandatory fields to enable authentication with GitHub
```

Comment:
I believe we should always repeat, for clarity, if we talk about the file or the OpenShift object (config map):

```suggestion
.`{my-app-config-file}` file fragment with mandatory fields to enable authentication with GitHub
```